x/tx/signing/textual/internal/testdata/encode.json

```diff
@@ -1,19 +1,19 @@
 [
     {
         "screens": [],
-        "encoding": "80"
+        "encoding": "a10180"
```

Comment:
All CBOR encodings have `a101` prepended. Looking at the [RFC](https://datatracker.ietf.org/doc/rfc8949/) Appendix B, this makes sense:
>      | 0xa0..0xb7 | map (0x00..0x17 pairs of data items follow)    |

- `a1`: map with 1 item
- `01`: key of first item